### PR TITLE
Debug anchore grype pipeline

### PR DESCRIPTION
**Pull request by Sidbhasin13**

### CODE REVIEW

File: Dockerfile
Comment (RAJANAGORI) [inline Dockerfile:1]: @Sidbhasin13 this change should be done in the programming dockefile as that image is considered as the base image of the Nightingale's final docker image. 
For for information, Please refer to the architecture diagram in the wiki link. Let me know your doubts.

**Review by RAJANAGORI — CHANGES_REQUESTED** [review]
@Sidbhasin13 Thanks for the changes but there is a request to add the config of the Dockerfile in the programming image file under the Dockerfiles folder.

File: Dockerfile
Comment (Sidbhasin13) [reply Dockerfile:1]: @RAJANAGORI made the changes as per your suggestions.

**Review by RAJANAGORI — APPROVED** [review]

**Comment by RAJANAGORI** [comment]
Changes Merged !!

**Pull request merged**